.gitignore

```diff
@@ -174,4 +174,8 @@ cython_debug/
 .pypirc
 tmp*
 logs/*
-.vscode
\ No newline at end of file
+.vscode
+
+# Temporary files
+tmp/*
+prompt_dataset.json
```

Comment:
We can make the output path as an argument to the `prompt_set` entry to avoid this specific ignore rule.